.changeset/cold-cups-buy.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/catalog-model': minor
```

Comment:
Realized there's no point of this changeset, there's nothing to be released right?

Reply:
I guess not. Also, crap, gotta rebase (or reconstruct) again